.gitignore

```diff
@@ -4,5 +4,5 @@ storybook-static
 .idea
 esm
 cjs
-index.d.ts
```

Comment:
This makes `src/typings/index.d.ts` searchable in IDEs that support ignoring files from `.gitignore`. Perhaps here due to historical reasons and could be removed entirely?